src/Microsoft.Management.Deployment/PackageManager.idl

```diff
@@ -1233,6 +1433,20 @@ namespace Microsoft.Management.Deployment
         /// (Installing, Installed) together at the same time.
         PackageCatalogReference CreateCompositePackageCatalog(CreateCompositePackageCatalogOptions options);
 
+        [contract(Microsoft.Management.Deployment.WindowsPackageManagerContract, 11)]
+        {
+            Windows.Foundation.IAsyncOperationWithProgress<AddPackageCatalogResult, AddPackageCatalogProgress> AddPackageCatalogAsync(AddPackageCatalogOptions options);
+
+            /// Removes a source from the Windows Package Manager source list.
+            Windows.Foundation.IAsyncOperationWithProgress<RemovePackageCatalogResult, RemovePackageCatalogProgress> RemovePackageCatalogAsync(String catalogName);

```

Comment:
I would strongly consider making this (and the other non-Add methods) a member of `PackageCatalogReference`.

Reply:
We decided to keep it part of PackageManger class in the API review.